zkg.config

```diff
@@ -66,7 +66,7 @@ bin_dir =
 
 # The directory containing Zeek distribution source code.  This is only
 # needed when installing packages that contain Zeek plugins that are
-# not pre-built.  The legacy name of this option is "bro_dist".
+# not pre-built.
```

Comment:
Today `zeek_dist` is legacy with regards to zkg and an install tree works fine for everything. That's my understanding. I'm unsure what "Zeek plugins that are note pre-built" refer to in fact.

Maybe we can note down that zeek_dist isn't needed today outside of maybe dev users?

Reply:
Added this to the comment:

> This value is generally not needed by most users other than plugin developers anymore.